feeder/feeder.py

```diff
@@ -20,10 +20,12 @@
 from opbeat.handlers.logging import OpbeatHandler
 if 'OPBEAT_SECRET' in os.environ:
     opbeat_client = opbeat.Client(
-        app_id='c36849e44e',
-        organization_id='093c53b0da9d43c4976cd0737fe0f2b1',
-        secret_token=os.environ['OPBEAT_SECRET'])
-else: opbeat_client = None
```

Comment:
As I said, in my instance it worked but if you like to have it, it is no problem to insert.